internal/cli/atlas/onlinearchive/online_archive_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build unit
+
+package onlinearchive
+
+import (
+	"testing"
+
+	"github.com/mongodb/mongocli/internal/test"
+)
+
+func TestBuilder(t *testing.T) {
+	test.CmdValidator(
+		t,
+		Builder(),
+		7,
+		[]string{},
+	)
+}
```

Comment:
Could we add the `Test"Command"Builder `test also to the other commands?

Reply:
if you insist, making this PR a bit larger tho